libraries/botbuilder-schema/botbuilder/schema/teams/teams_channel_account.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+
+from botbuilder.schema import ChannelAccount
+
+
+class TeamsChannelAccount(ChannelAccount):
+    def __init__(
+        self,
+        id="",
+        name="",
+        aad_object_id="",
+        role="",
+        given_name="",
+        surname="",
+        email="",
+        userPrincipalName="",
```

Comment:
Is "userPrincipalName" needed?  Why not "user_principal_name"?

Reply:
See comment below